raqm.c

```diff
@@ -573,7 +581,6 @@ raqm_shape_u32 (unsigned int* u32_str,
         }
         else
         {
-            int j;
```

Comment:
Hmm, I though a declaration on top of a code block is valid in C89, what was the exact warning?